app.py

```diff
@@ -0,0 +1,251 @@
+#!/usr/bin/env python
+import os
+import random
+"""
+    This module implements Amity as a class based interface for the entire
+    room allocation system
+"""
```

Comment:
maybe not have comments in between imports.. maybe put the comments at the very top..